### PR TITLE
create change_status action

### PR DESCRIPTION
**Pull request by nooralnabi**
please check the action, sorry for delete last PR because a misunderstand what ail side by change column name from active to status

### CODE REVIEW

File: app/controllers/contractors_controller.rb [hunk 1]
```diff
@@ -12,6 +12,14 @@ def index
   def show
   end
 
+  def changes_status
+  @contractor = Contractor.find( params[:id])
+  @contractor.active =!@contractor.active
```
Comment (ssharba) [inline app/controllers/contractors_controller.rb]: @nooralnabi Good!

File: config/routes.rb [hunk 2]
```diff
@@ -2,7 +2,11 @@
 
   get 'public/index'
 
-  resources :contractors
+  resources :contractors do
+    member do
+       post 'changes_status'
```
Comment (ssharba) [inline config/routes.rb]: change_status not changes_status

Comment (nooralnabi) [inline config/routes.rb]: ok I changed it

**Comment by aqattan** [comment]
@nooralnabi Please fix the switch button, it should appear when i click on Contractors, not after i refresh the page.

**Pull request closed without merging**